rllib/utils/policy.py

```diff
@@ -185,13 +191,38 @@ def local_policy_inference(
         policy.agent_connectors
     ), "policy_inference only works with connector enabled policies."
 
+    # TODO(Artur): Remove this after we have migrated deepmind style preprocessing into
+    #  connectors (and don't auto-wrap in RW anymore)
+    if any(
+        [
+            o.shape == (210, 160, 3) if isinstance(o, np.ndarray) else False
```

Comment:
is there a CONSTANT defined for this (210, 160, 3) tuple somewhere?

Reply:
You know there is 😄

Reply:
Done